backend/spec/views/spree/admin/shared/navigation_footer_spec.rb

```diff
@@ -0,0 +1,41 @@
+require 'spec_helper'
+
+describe "spree/admin/shared/_navigation_footer", type: :view do
+  let(:user) { create(:admin_user) }
```

Comment:
is it possible for an ordinary Spree::User to be viewing the admin backend at all? I guess it doesn't matter for a view test though. 

I believe it does need to be a saved user (with an id) (or something mocked similarly), so generating a path to the admin user edit page for the user will work. 

I am wary of mocking too much, because of other spree/solidus tests that mock so much they become unreliable. 

I think it's prob okay the way it is, most 'realistic' test scenario, but if you require a different approach here just let me know. I suppose it can be `create(:user)`, or a built non-saved user with it's `id` mocked out so a path can still be generated.... that latter one is not attractive to me for several reasons though.